Library/Formula/dlite.rb

```diff
@@ -0,0 +1,39 @@
+class Dlite < Formula
+  desc "The simplest way to use Docker on OSX"
+  homepage "https://github.com/nlf/dlite"
+  url "https://github.com/nlf/dlite/archive/1.1.3.tar.gz"
+  sha256 "a87c4d7b7cd666082dc0fc44db2ca2b03efb4ee66676450c4844e7de0420d8e6"
+
+  # DLite depends on the Hypervisor framework which only works on
+  # OSX versions 10.10 (Yosemite) or newer
+  depends_on :macos => :yosemite
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+
+    path = buildpath/"src/github.com/nlf/dlite"
+    path.install Dir["*"]
+
+    cd path do
+      system "make", "dlite"
+      bin.install "dlite"
+    end
+  end
+
+  def caveats
+    <<-EOS.undent
+
+      Installing and upgrading dlite with brew does not automatically
+      install or upgrade the dlite daemon and virtual machine.
+      For more information please refer to:
+        https://github.com/nlf/dlite
```

Comment:
Can remove the URL and `For more information please refer to` lines; people will know to look at the homepage if they need more information and it's documented in the formula already.